PopupBridge/src/main/java/com/braintreepayments/api/PopupBridgeClient.kt

```diff
@@ -181,9 +188,9 @@ class PopupBridgeClient @SuppressLint("SetJavaScriptEnabled") @VisibleForTesting
     companion object {
         // NEXT MAJOR VERSION: consider using a `com.braintreepayments...` prefixed request key
         // to prevent shared preferences collisions with other braintree libs that use browser switch
```

Comment:
Is this still applicable?

Reply:
Hmm I don't think it still applies, but I don't think I have full context on this. Do you know if SharedPreferences use to be used within browser switch to store the pending intent?

Reply:
Ah yes I think it used to be - might no longe be relevant